### PR TITLE
implement attachment block for files >5MB

### PR DESCRIPTION
**Pull request by mixmix**
selection_387

This is a straight port from my patchbay work. (apart from the hot pink, which is more legible)

Pretty simple

### CODE REVIEW

File: modules/message/html/compose.js [hunk 1]
```diff
@@ -53,7 +53,26 @@ exports.create = function (api) {
...
+    )
     var fileInput = api.blob.html.input(file => {
+      const megabytes = file.size / 1024 / 1024
+      if (megabytes >= 5) {
+        const rounded = Math.floor(megabytes*100)/100
+        warningMessage.set([
+          h('i.fa.fa-exclamation-triangle'),
+          h('strong', file.name),
+          ` is ${rounded}MB - the current limit is 5MB`
+        ])
```
Comment (mixmix) [inline modules/message/html/compose.js]: Ideally we'd get sbot to rm that file ... https://scuttlebot.io/apis/scuttlebot/blobs.html#rm-async

File: modules/message/html/compose.js [hunk 2]
```diff
@@ -53,7 +53,26 @@ exports.create = function (api) {
...
+    var warningMessage = Value(null)
+    var warning = h('section.warning',
+      { className: when(warningMessage, '-open', '-closed') },
+      [
+        h('div.warning', warningMessage),
+        h('div.close', { 'ev-click': () => warningMessage.set(null) }, 'x')
+      ]
+    )
     var fileInput = api.blob.html.input(file => {
+      const megabytes = file.size / 1024 / 1024
```
Comment (ahdinosaur) [inline modules/message/html/compose.js]: this should probably be a function `bytesToMegabytes`

File: modules/message/html/compose.js [hunk 3]
```diff
@@ -53,7 +53,26 @@ exports.create = function (api) {
...
+      if (megabytes >= 5) {
+        const rounded = Math.floor(megabytes*100)/100
```
Comment (ahdinosaur) [inline modules/message/html/compose.js]: i'm confused what this does. does this round a decimal number to two digits?

Comment (mixmix) [reply modules/message/html/compose.js]: yep, is there a tidier way ?

**Comment by mixmix** [comment]
Good enough to merge ?

**Pull request merged**